python/ray/serve/tests/test_telemetry.py

```diff
@@ -640,5 +644,52 @@ def check_telemetry(tag_should_be_set: bool):
         wait_for_condition(check_telemetry, tag_should_be_set=True)
 
 
+def test_multiplexed_detect(manage_ray):
+    """Check that multiplexed api is detected by telemetry."""
+
+    subprocess.check_output(["ray", "start", "--head"])
+    wait_for_condition(check_ray_started, timeout=5)
+
```

Comment:
please add a check here that the multiplexed usage tag is _not_ present prior to using it

Reply:
added